website/source/docs/templates/communicator.html.md

```diff
@@ -75,8 +75,9 @@ The SSH communicator has the following options:
 -   `ssh_bastion_port` (number) - The port of the bastion host. Defaults to
     `22`.
 
--   `ssh_bastion_private_key_file` (string) - A private key file to use to
-    authenticate with the bastion host.
+-   `ssh_bastion_private_key_file` (string) - Path to a PEM encoded private
```

Comment:
Is this a good suggestion @rickard-von-essen ?
```suggestion
-   `ssh_bastion_private_key_file` (string) - Path to a private
```

Reply:
Also related, since this check was added, then it probably needs to be PEM encoded as `FileSigner` checks for this
 

https://github.com/hashicorp/packer/blob/0b68045848941d4ee9cea28679c830e83a4e9753/helper/communicator/config.go#L281-L293

Reply:
@azr this is proper validation, because `SSHBastionPrivateKeyFile` would be parsed later with `helperssh.FileSigner` - https://github.com/hashicorp/packer/blob/v1.3.2/helper/communicator/step_connect_ssh.go#L228-L235
`ssh.ParsePrivateKey` also requires PEM encoded private key - https://godoc.org/golang.org/x/crypto/ssh#ParsePrivateKey

Reply:
Right, then all is good 🙂